comms/AttributeType.cpp

```diff
@@ -250,8 +253,10 @@ static CacheT<std::_tstring, CAttributeType> AttributeTypeCache;
 
 IAttributeType *AttributeTypeFromExtension(const std::_tstring & extension) {
     IAttributeType *attrType = NULL;
-    if (boost::algorithm::iequals(extension, ATTRIBUTE_TYPE_ESDL) || boost::algorithm::iequals(extension, ATTRIBUTE_TYPE_ECM))
+    if (boost::algorithm::iequals(extension, ATTRIBUTE_TYPE_ESDL))
         attrType = CreateIAttributeESDLType();
+    else if (boost::algorithm::iequals(extension, ATTRIBUTE_TYPE_ECM))
+        attrType = CreateIAttributeECMType();
```

Comment:
I thought we were trying to not use the ECM type now?

Reply:
@GordonSmith I did think a lot about this and in my changes to treat them the same, I thought it cleaner to keep them as separate types but in the end, treat them the same. The problem is having an extension and a type that doesn't match. I think this cleaner.